### PR TITLE
handling of redirect event

### PR DESCRIPTION
**Pull request by no-chris**
Related to https://oat-sa.atlassian.net/browse/TAO-5698

Depends on:
- [x] https://github.com/oat-sa/generis/pull/546
- [ ] https://github.com/oat-sa/extension-tao-backoffice/pull/63

 This PR let's you link a resource from the task queue component. If the resource has been deleted in the meantime a error message is displayed.

 How to test : 
 - Install the extension taoTaskQueue
 - Set up the task queue `sudo -u www-data php index.php 'oat\taoTaskQueue\scripts\tools\InitializeQueue' --broker=rds --persistence=default`
 -Keep running the worker ` sudo -u www-data php index.php 'oat\taoTaskQueue\scripts\tools\RunWorker'
 - Try to import tests
 - Open the import ed test from another  extension page
 - Redo it but try to remove the imported test in the meantime
 - Run the test `/tao/views/js/test/core/taskQueue/test.html`

### CODE REVIEW

**Comment by no-chris** [comment]
Please check the bump version

**Comment by ssipasseuth** [comment]
Strangely I always have the 202 error when I try to go to the resource even when it exists. I tested it on a qti test import.
The icon is style the preview one. Could the compagnon https://github.com/oat-sa/extension-tao-task-queue/pull/67 missing new commits ?

**Comment by krampstudio** [comment]
@ssipasseuth yes, I forgot to add some files to both PRs. Could you please check again.
@martijn-tao I did what I could since the original PHP code isn't commented and I have no clue what it does...

File: manifest.php [hunk 1]
```diff
@@ -172,9 +172,9 @@
     ),
     'constants' => array(
         #TAO version number
-        'TAO_VERSION' => '3.3.0-sprint91',
+        'TAO_VERSION' => '3.3.0-sprint90',
```
Comment (jbout) [inline manifest.php]: downgrade of sprint seems to be a merge problem, please check if there are others

File: models/classes/taskQueue/TaskLog/Decorator/RedirectUrlEntityDecorator.php [hunk 2]
```diff
@@ -22,16 +22,27 @@
 
 use oat\tao\model\accessControl\AclProxy;
 use oat\tao\model\taskQueue\TaskLog\Entity\EntityInterface;
+use oat\tao\model\taskQueue\TaskLogInterface;
 use oat\taoBackOffice\controller\Redirector;
 
 /**
  * @author Gyula Szucs <gyula@taotesting.com>
  */
 class RedirectUrlEntityDecorator extends TaskLogEntityDecorator
```
Comment (jbout) [inline models/classes/taskQueue/TaskLog/Decorator/RedirectUrlEntityDecorator.php]: No PRs should be merged anymore without relevant unit tests

Comment (krampstudio) [reply models/classes/taskQueue/TaskLog/Decorator/RedirectUrlEntityDecorator.php]: Indeed we will find someone to write them

Comment (siwane) [reply models/classes/taskQueue/TaskLog/Decorator/RedirectUrlEntityDecorator.php]: @krampstudio , it is done

**Comment by krampstudio** [comment]
@jbout @ssipasseuth backend unit tests have been added, could you please review it again

**Pull request merged**